react/ecmascript/02 - promises/index.js

```diff
@@ -0,0 +1,49 @@
+// Hint: use setInterval, create a new Promise and measure time with Date.now()
+
+export function delay(time) {
+
+var start = Date.now();
+
+return new Promise(
+    function (resolve, reject) {
+      if(time >= 400 && time <= 600) {
```

Comment:
Define consts with this values. For example `const MIN_BASE_TIME = 400;` `const MAX_BASE_TIME = 600;`